web/src/test/java/com/javaholics/web/service/EventServiceTest.java

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
testy zakomentowane, bo sie nie kompiluja.
Prosze nie commitujcie niekompilujacego sie kodu, bo wtedy ktos kto go sciaga nie moze odpalic aplikacji - czy to wewnatrz zespolu czy z zewnatrz, jak w tym przypadku ja.